.storybook/package.json

```diff
@@ -76,7 +76,7 @@
 		"react-syntax-highlighter": "^15.6.1",
 		"remark-gfm": "^4.0.0",
 		"rollup-plugin-postcss-lit": "^2.1.0",
-		"storybook": "^8.4.7",
+		"storybook": "^8.6.12",
```

Comment:
Looks like a Storybook upgrade got added in here. This newer version still causes all Docs page descriptions to disappear.

Reply:
Ah thank you, yes!

Reply:
Pushed up that roll-back